app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="main-signup-container">
+<div class="signup-container">
+<h2 class="signup">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email, class:"name-label" %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"name-input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password, class:"name-label" %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class:"name-input" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class:"signup-button" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+
+</div>
```

Comment:
- [ ] I will encourage you to style your login according to the wireframe so that it would meet the requirement.

### Your design
![image](https://github.com/sanieni6/BlogApp/assets/86200356/57696dee-a25a-4aee-913c-f80bd7a86da5)

### Wireframe
![image](https://github.com/sanieni6/BlogApp/assets/86200356/8b358a44-8eb2-4248-ae4a-f4e6d64c270d)